resources/wikia/libraries/visit-source/visit-source.js

```diff
@@ -0,0 +1,45 @@
+var VisitSource = (function () {
+	/**
+	 *
+	 * @param cookieName - cookie where session referrer is stored
+	 * @param cookieDomain - cookie domain parameter - for saving cookies
+	 * @param isSession - flag, based on value cookie is stored only for current session or lifetime
+	 */
+	function VisitSource(cookieName, cookieDomain, isSession) {
+		if (isSession === void 0) { isSession = true; }
```

Comment:
This is how .ts gets compiled to .js
https://github.com/Wikia/visit-source/blob/master/src/visit-source.ts#L19